src/grpcConnectionPool.ts

```diff
@@ -22,8 +22,11 @@ const seedNodes = {
 
 export default class GRPCConnectionPool {
   channels: Channel[]
+  utils: UtilsController
+
+  constructor (network: 'testnet' | 'mainnet', utils: UtilsController, dapiUrl?: string) {
```

Comment:
You don't have to add UtilsController as an argument here, move out getRandomItem() function in a separate file and import directly instead